Source/Core/VideoBackends/OGL/TextureCache.cpp

```diff
@@ -113,8 +80,7 @@ TextureCache::~TextureCache()
 
   if (g_ActiveConfig.backend_info.bSupportsPaletteConversion)
   {
-    s_palette_stream_buffer.reset();
```

Comment:
I think it is fine here to swap the order of destructing those objects. But for the record, this here is a behavior change.

Reply:
Indeed. The only reason this was likely here is because it can't be done automatically as a file-static variable.